Signal/src/network/PushManager.m

```diff
@@ -193,6 +193,7 @@ - (void)application:(UIApplication *)application
 
             [ThreadUtil sendMessageWithText:replyText
                 inThread:thread
+                quotedMessage:nil // TODO we could do an actual quoted reply here, but most apps don't.
```

Comment:
* Let's check in with Myles.
* Please put param/argument comments on the line before them.

Reply:
updated. I'll add this to the list of things to go over with myles.